plotbitrate.py

```diff
@@ -342,11 +342,15 @@ def try_get_frame_time_from_node(node: eTree.Element) -> Optional[float]:
 def create_progress(duration: int):
     # set to negative, so 0% gets reported
     last_percent = -1.0
+    offset = -1
 
     def report_progress(frame: Optional[Frame]):
         nonlocal last_percent
+        nonlocal offset
         if frame:
-            percent = round((frame.time / duration) * 100.0, 1)
+            if offset == -1:
+                offset = frame.time
+            percent = round(((frame.time-offset) / duration) * 100.0, 1)
```

Comment:
```suggestion
            percent = round(((frame.time - offset) / duration) * 100.0, 1)
```

Reply:
Since I haven't seen a file like this before I'm assuming `duration` is reduced as well? Let's consider an example where you have a 100s video and the start time is 20s. If `duration` was still 100s you'd reach 80s at the end, which is not 100% because of the 20s `offset`. If `duration` is corrected by ffmpeg to 80s (100s - 20s) then it should be fine. Depending on which of these two scenarios you're observing we may need to subtract `offset` from `duration` as well.

Reply:
so as an example some of my files are recordings, others have been generated with gstreamer, one example file if I dump the xml:
```
<ffprobe>
    <frames>
        <frame pkt_pts_time="3600.000000" best_effort_timestamp_time="3600.000000" pkt_size="229" pict_type="I" />
        ...
        <frame pkt_pts_time="3631.000000" best_effort_timestamp_time="3631.000000" pkt_size="76213" pict_type="I" />
    </frames>

    <format filename="omx_test2.ts" nb_streams="1" nb_programs="1" format_name="mpegts" format_long_name="MPEG-TS (MPEG-2 Transport Stream)" start_time="3600.016656" duration="31.000011" size="12285236" bit_rate="3170382" probe_score="50" />
</ffprobe>
```
so the output from ffprobe is correctly compensated already/